requirements/edx/base.txt

```diff
@@ -90,7 +90,7 @@ python-dateutil==2.1
 # We can also remove the fix to auth_url in third_party_auth/saml.py when that fix is included upstream.
 python-social-auth==0.2.12
 
-pytz==2015.2
+pytz>=2016.7
```

Comment:
FYI @edx/devops @nedbat  We are un-pinning the `pytz` library used on edx-platform and a few other repos.  So it will be automatically updated whenever there's a new version and pre-reqs are installed.  The rationale is that we are using this library only for its database of timezones and no code other than getting UTC.  Unpinning the version saves us from repeated busy work and allows us to automatically support any updated timezones in the library.

More discussion on this (yesterday) in the "T&L - Beryl" hipchat room.